test/error.test.js

```
@@ -117,11 +129,9 @@ describe("Test error", async function () {
     });
 
     it("should test shouldThrow", async function () {
-        const error = { code: -32003 };
-        assert(shouldThrow(error));
-
-        error.code = 3;
+        const error = { code: -32003, message: "execution reverted" };
         assert(shouldThrow(error));
+        delete error.message;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)